### DISCUSSION TITLE
Grid2 Issue

### DISCUSSION THREAD

**Discussion by ghost** [SFX-WoW/Masque#174] [Issues] [answered]
Masque overrides my grid2 to give it a frame, which alters its location for each raid frame. Its very frustrating.

When I go into masque, it doesnt say its utilising it, only elvui and weakauras. So I have no idea how to make it drop the action, except to turn it off at global.

It shouldn't be overriding a random addon if its unable to put it into its dropdown to turn it off.

Could you please add the override for masque to turn off its layer for grid2's player frames please :).

**Comment by StormFX** [comment, marked as answer]
Masque doesn't affect Grid2.
